### DISCUSSION TITLE
Customized compile function

### DISCUSSION THREAD

**Discussion by Ahacad** [micromark/micromark#76] [Ideas] [answered]
Hi, I'm trying to convert a markdown file to a LaTeX file. I found the default transformation by pandoc cannot fit my needs, and I think maybe it would be ok to write a customized compile function for micromark so I can compile mdAST to LaTeX string, do you think this would be feasible?

**Comment by wooorm** [comment, marked as answer]
That is feasible. But it’s much easier to use the syntax tree: https://github.com/micromark/micromark#syntax-tree.
